generate-templates.yaml

```diff
@@ -121,6 +121,23 @@
       osinfoname: "{{ oslabels[0] }}"
       cloudusername: cloud-user
 
+  - name: Generate RHEL 6 templates with a legacy machine type pc-i440fx
+    template:
+      src: templates/contrib/pc/rhel6.tpl.yaml
```

Comment:
@ksimon1 we want this template, but we do not want to publish it as part of the releases, how can we do this?

Reply:
Add command
`rm dist/templates/<Template name>`
here https://github.com/kubevirt/common-templates/blob/master/makefile#L26